app/src/main/java/com/es/findsoccerplayers/Utils.java

```diff
@@ -23,5 +35,24 @@ static void showSuccessResetPswToast(Context c){
         Toast.makeText(c, R.string.reset_psw_success, Toast.LENGTH_SHORT).show();
     }
 
+    static void storeUserInfoDB(final String TAG, FirebaseAuth fAuth, FirebaseFirestore fStore, String name, String surname, String date) {
```

Comment:
io non lo metterei in Utils. E' qualcosa che fa la activity register, quindi è un suo metodo privato

Reply:
Per adesso, poi quando dovremmo registrare anche le credenziali di chi si logga con google, non si passa per l'activity register, perciò serve a più classi